samples/debug/ppi_trace/Kconfig

```diff
@@ -0,0 +1,67 @@
+# Kconfig - Config options for PPI trace sample
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+mainmenu "PPI trace sample"
+
+menu "PPI trace pins configuration"
+
+config PPI_TRACE_PIN_RTC_COMPARE_EVT
+	int "Trace pin for RTC Compare event"
+	default 17 if BOARD_NRF9160_PCA10090
+	default 1 if BOARD_NRF51_PCA10028
+	default 3
+	help
+	  Pin is toggled when an RTC Compare event occurs.
+
+config PPI_TRACE_PIN_RTC_TICK_EVT
+	int "Trace pin for RTC Tick event"
+	default 18 if BOARD_NRF9160_PCA10090
+	default 2 if BOARD_NRF51_PCA10028
+	default 4
+	help
+	  Pin is toggled when an RTC Tick event occurs.
+
+config PPI_TRACE_PIN_LFCLOCK_STARTED_EVT
+	int "Trace pin for LFCLOCK Started event"
+	default 19 if BOARD_NRF9160_PCA10090
+	default 3 if BOARD_NRF51_PCA10028
+	default 28
+	help
+	  Pin is toggled when an LFCLOCK Started event occurs.
+
+config USE_BLUETOOTH_RADIO_EVENTS
+	bool
+	default y
+	select BT
+	depends on SOC_SERIES_NRF52X
+
+if USE_BLUETOOTH_RADIO_EVENTS
+
+config PPI_TRACE_PIN_RADIO_ACTIVE
+	int "Trace pin for radio activity"
+	default 4 if BOARD_NRF51_PCA10028
```

Comment:
This does not make much sense, since you cannot enable Bluetooth radio events on nRF51 Series.

Reply:
removed.